src/main/java/com/synopsys/integration/alert/channel/azure/boards/service/AzureCustomFieldManager.java

```diff
@@ -128,8 +128,6 @@ public void installCustomFields(String projectName, String workItemTypeName) thr
     private Optional<ProjectWorkItemFieldModel> getAlertCustomProjectField(String projectName, String fieldReferenceName) {
         try {
             return Optional.of(projectService.getField(organizationName, fieldReferenceName));
-        } catch (IOException e) {
```

Comment:
Is this not thrown? why are we removing it?

Reply:
It isn't thrown anymore.